samples/bluetooth/mesh/light_ctrl/src/model_handler.c

```diff
@@ -101,6 +102,23 @@ static const struct bt_mesh_health_srv_cb health_srv_cb = {
 	.attn_off = attention_off,
 };
 
+#ifdef CONFIG_BT_MESH_LARGE_COMP_DATA_SRV
```

Comment:
`ifdef` is not required

Reply:
Will remove if [that](https://github.com/nrfconnect/sdk-nrf/pull/16580/files#r1688171790) works. (Otherwise it will give compilation warnings when not using large comp data, I guess)